fms2_io/include/scatter_data_bc.inc

```diff
@@ -184,6 +188,8 @@ subroutine scatter_data_bc_3d(fileobj, varname, vdata, bc_info, unlim_dim_level)
   integer(kind=i8_kind) :: chksum_val !< Checksum value calculated from the read data
   character(len=32) :: chksum !< Cheksum value converted to a string
   character(len=32) :: chksum_read !< String checksum read in from file
+  logical :: chksum_ignore = .FALSE. ! local variable for data integrity checks
+                                     ! default: .FALSE. - checks enabled
```

Comment:
I've doxygenized the interface variable.  This is a local variable, I thought we only needed to document for readability, not doxygenize.

Reply:
I think it's best to follow the examples and rules of the style and documentation guides that FMS provides.
1. The documentation guide shows an example of a local variable being documented http://noaa-gfdl.github.io/FMS/md_docs_doxygenGuide.html
```fortran
subroutine foo()
  integer :: bar !< variable description
```
2. In the [general section of the style guide](https://github.com/NOAA-GFDL/FMS/blob/main/CODE_STYLE.md#general) it says to " Follow coding style of the current file, as much as possible." All of the other variables are documented with a doxygen style comment.
3. While doxygen may not currently parse out the local variables documented in this way, if it is updated to read these comments, then someone won't have to spend a month searching through the code to add `<` and `!` all over the place.  I think it's prudent to have the code prepared in this way.

Reply:
That guide should be updated so `bar` is an argument instead, because it shouldn't really be required for every local variable, just args and in types. But in this case we might as well match the style of the rest of the file, since the other local vars are that way already.